docs/installation.md

```diff
@@ -14,6 +14,24 @@ pip install improv --no-binary pyzmq
 to build `pyzmq` from source if you're running into ZMQ errors.
 ````
 
+## Required dependencies
+
+### Redis
+
+_improv_ uses Redis, an in-memory datastore, to hold data to be communicated between actors. _improv_ has been tested with Redis server version 7.2.4. Please refer to the instructions below for your operating system:
+
+#### macOS
+A compatible version of Redis can be installed via Homebrew:
+```
+brew install redis
+```
+
+#### Linux
+A compatible version of Redis can be installed for most standard Linux distributions by following Redis' instructions on their [installation guide](https://redis.io/docs/latest/operate/oss_and_stack/install/install-redis/install-redis-on-linux/).
```

Comment:
Maybe give an example? Something like "On Ubuntu, for instance, you can just
```
apt-get install redis
```

Reply:
The `apt-get install` command can only be done after following their instructions to add the Redis GPG key to your keyring. I added a note to indicate that the install process is short, but in case they change it later I thought it might not be best to immortalize their current install process in our docs.